gdal/port/cpl_swift.cpp

```diff
@@ -106,33 +107,46 @@ bool VSISwiftHandleHelper::GetConfiguration(CPLString& osStorageURL,
         return true;
     }
 
-    CPLString osAuthURL = CPLGetConfigOption("SWIFT_AUTH_V1_URL", "");
-    CPLString osUser = CPLGetConfigOption("SWIFT_USER", "");
-    CPLString osKey = CPLGetConfigOption("SWIFT_KEY", "");
-    if( osAuthURL.empty() || osUser.empty() || osKey.empty() )
+    CPLString osAuthV3URL = CPLGetConfigOption("SWIFT_AUTH_V3_URL", "");
```

Comment:
this new config option (and possible other new ones) should be documented in doc/source/user/virtual_file_systems.rst

Reply:
I will update the documentation once config options will be settled.